source/direct_debit/index.html.md.erb

```diff
@@ -241,6 +241,13 @@ You could then get a single page of these mandates by running:
 
 This would return payments 501 to 1000.
 
+## Test a mock bank account
+
+To test confirming bank account details as a user, use the following with your test account:
```

Comment:
Not sure “confirming” is the right word here.

Reply:
Apologies, something went wrong here... this should be "entering". I'll fix now.